src/main/resources/static/style.css

```diff
@@ -0,0 +1,21 @@
+.content {
+    width: 100%;
+    max-width: 1000px;
+    margin: auto;
+}
+
+.header {
+    width: 100%;
+    background-image: url('/recipe.jpg');
```

Comment:
zastanawia mnie dlaczego nei wyświetla się obrazek. Czy to kwestia źle dobranego obrazka? :D

Reply:
U mnie tego na pewno nie pokaze poniewaz w repozytorium aplikacji nie ma tego obrazka. Spróbuj dorzucić go do katalogu `resources`